src/test/kotlin/qbit/StorageTest.kt

```diff
@@ -39,4 +43,43 @@ class StorageTest {
         assertEquals(setOf(subNs["sub-data"]), storage.keys(subNs).toSet())
     }
 
+    @Test
+    fun testSwapHead() {
+        val user = Namespace("user")
+        val _id = ScalarAttr(user["val"], QString)
+
+        val root = Files.createTempDirectory("qbit").toFile()
+        val storage = FileSystemStorage(root)
+
+        val conn = qbit(storage)
+        conn.persist(_id)
+
+        val e = Entity(_id eq "1")
+        val trx = conn.trx()
+        trx.persist(e)
```

Comment:
@S1x6 Is it essential for mutant elimination to persist via explicit transaction, but not implicit transaction within conn.persist?

Reply:
Changed to implicit transaction